details/operator-sdk/config/unified-installer-script/kustomization.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namePrefix: operator-
+resources:
+  - ../crd
+  - ../rbac
+  - ../manager
```

Comment:
@nhammern I did end up creating a new folder with a kustomization file otherwise I would've had to either add `namePrefix: operator-` to the cluster-install kustomization (which would've screwed up the neptune.yaml names) or modified the install script to take into account the lack of the prefix.